evm_body.yaml

```diff
@@ -834,12 +834,57 @@ alchemy_getTokenBalances:
             3. Options (optional) - An object that contains the following settings **[omitted below]**:
                 1. `pageKey`: Applies only to the `erc20` request type. A string address used for pagination. If more results are available, a `pageKey` will be returned in the response.
                 2. `maxCount`: Applies only to the `erc20` request type. Specifies the maximum count of token balances to return per call. This value defaults to 100 and is currently capped at 100.
-          minItems: 1
-          maxItems: 2
+          minItems: 2
+          maxItems: 3
           items:
-            type: string
-          default: ['0x95222290DD7278Aa3Ddd389Cc1E1d165CC4BAfe5', 'erc20']
-
+            oneOf:
```

Comment:
@SahilAujla do we need this oneOf?

Reply:
Yes @elanh we need this to bypass Readme's limitation of not displaying multi-type arrays correctly. 

If we don't use it, it will display all the params as strings which is incorrect. 

But when we use this, it says that the params array can have items of either of these types: 

- a string
- a string or an array
- an object

but this also results in displaying repeated options like this: 
![image](https://github.com/alchemyplatform/docs-openapi-specs/assets/83442423/4cc21e4f-f33e-466b-8610-86bdc8524c2f)

but that's just the way to workaround it.

Since our composer can display it correctly, we don't use this `oneOf` there, so we remove that in `prefixItems` for the composer.